features/configuration_loading.feature

```diff
@@ -11,51 +11,54 @@ Feature: Offer different ways how to load configuration
   And finally, it should check your HOME directory.
 
   Scenario: No configuration
-    When I run reek spec/samples/configuration_loading/minimal_dirty.rb
+    Given a minimal dirty file called 'minimal_dirty.rb'
+    When I run reek minimal_dirty.rb
     Then the exit status indicates smells
     And it reports:
       """
-      spec/samples/configuration_loading/minimal_dirty.rb -- 3 warnings:
+      minimal_dirty.rb -- 3 warnings:
         [1]:C has no descriptive comment (IrresponsibleModule)
         [1]:C has the name 'C' (UncommunicativeModuleName)
         [2]:C#m has the name 'm' (UncommunicativeMethodName)
       """
 
   Scenario: Configuration via CLI
-    When I run reek -c spec/samples/minimal_smelly_and_masked/config.reek spec/samples/minimal_smelly_and_masked/minimal_dirty.rb
+    Given a minimal dirty file called 'minimal_dirty.rb'
```

Comment:
Any chance we can unify "dirty.rb" and "minimal_dirty.rb"? Or  at least rename them?